apps/src/templates/rubrics/RubricContainer.jsx

```diff
@@ -103,6 +104,47 @@ export default function RubricContainer({
     fetchAiEvaluations();
   }, [fetchAiEvaluations]);
 
+  const [allTeacherEvaluationData, setAllTeacherEvaluationData] = useState([]);
+
+  const fetchTeacherEvaluationAll = (rubricId, sectionId) => {
+    return fetch(
+      `/rubrics/${rubricId}/get_teacher_evaluations_for_all?section_id=${sectionId}`
+    );
+  };
+
+  useEffect(() => {
+    const abort = new AbortController();
+    if (!!rubricId && !!sectionId) {
+      fetchTeacherEvaluationAll(rubricId, sectionId).then(response => {
+        if (response.ok) {
+          response.json().then(data => setAllTeacherEvaluationData(data));
+        }
+      });
+    }
+    return () => abort.abort();
```

Comment:
What is being aborted here?

I admit I haven't used `AbortController` before. The example at https://developer.mozilla.org/en-US/docs/Web/API/AbortController/AbortController seems to indicate that if you want the `fetch` call above to be connected to the `AbortController`, you need to pass `abort.signal` as part of the `options` object passed to `fetch`.